docs/userGuide/syntax/badges.md

```diff
@@ -1,5 +1,19 @@
 ## Badges
 
+Example:
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">html</variable>
+<variable name="code">
+<span class="badge bg-primary">
+Some Plain Text
+</span>
+
+<span class="badge bg-primary">
+<md>**Some Markdown**</md>
```

Comment:
Sorry minor point but the bolding isn't obvious in the badge so maybe italics should be more obvious :)
```suggestion
<md>_Some Markdown_</md>
```